Frameworks/include/WYPopoverController.h

```diff
@@ -31,6 +31,10 @@
 
 #import "WYPopoverPrivate.h"
 
+#pragma clang diagnostic push
+#pragma clang diagnostic ignored "-Wnullability-completeness"
```

Comment:
that is the warning - you can look it up online.  i filed a bug: [https://github.com/Microsoft/WinObjC/issues/1645](https://github.com/Microsoft/WinObjC/issues/1645)

---
In reply to: [95002863](https://github.com/Microsoft/WinObjC/pull/1643#discussion_r95002863) [](ancestors = 95002863)